tests/test_model/test_fast_conv_bn_eval.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from unittest import TestCase
+
+import torch
+from mmcv.cnn import ConvModule
+from torch import nn
+
+from mmengine.model.fast_conv_bn_eval import \
+    turn_on_fast_conv_bn_eval_for_single_model
+from mmengine.testing import assert_allclose
+
+
+class BackboneModel(nn.Module):
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        conv0 = nn.Conv2d(6, 6, 6)
+        bn0 = nn.BatchNorm2d(6)
+        self.mod1 = ConvModule.create_from_conv_bn(conv0, bn0)
+        self.conv1 = nn.Conv2d(6, 6, 6)
+        self.bn1 = nn.BatchNorm2d(6)
+        self.conv2 = nn.Conv2d(6, 6, 6)
+        self.bn2 = nn.BatchNorm2d(6)
+        self.conv3 = nn.Conv2d(6, 6, 6)
+        self.bn3 = nn.BatchNorm2d(6)
+
+    def forward(self, x):
+        # this ConvModule can use fast_conv_bn_eval feature
+        x = self.mod1(x)
+        # this conv-bn pair can use fast_conv_bn_eval feature
+        x = self.bn1(self.conv1(x))
+        # this conv-bn pair cannot use fast_conv_bn_eval feature
+        # because `self.conv2` is used twice
+        x = self.bn2(self.conv2(self.conv2(x)))
+        # this conv-bn pair cannot use fast_conv_bn_eval feature
+        # because `self.bn3` is used twice
+        x = self.bn3(self.bn3(self.conv3(x)))
+        return x
+
+
+class TestFastConvBNEval(TestCase):
```

Comment:
torch.fx is a new feature of torch1.8+ so it is necessary to skip unit tests when torch<1.8.0. Refer to https://github.com/open-mmlab/mmengine/blob/b2295a258c4af3cdeb1a55fa5e49cddbd1c6c0bf/tests/test_model/test_wrappers/test_model_wrapper.py#L224